docs/writing-rules/private-rules.md

```diff
@@ -7,7 +7,7 @@ import MoreHelp from "/src/components/MoreHelp"
 
 # Private rules
 
-Users in the [Team and Enterprise tier](https://semgrep.dev/pricing) for Semgrep App can publish rules to the Semgrep Registry as private rules that are not visible to others outside their organization. The private rules enable you to hide code-sensitive information or legal requirements that prevent you from using a public registry.
+Private rules enable you to publish rules so only users in your organization can see them but these rules are not displayed externally. Users in the [Team and Enterprise tier](https://semgrep.dev/pricing) for Semgrep App can publish rules to the Semgrep Registry as private rules that are not visible to others outside their organization. The private rules enable you to hide code-sensitive information or legal requirements that prevent you from using a public registry.
```

Comment:
I'd remove this change, as you don't necessarily have to "publish" private rules

Reply:
Removing))